source/_integrations/emoncms.markdown

```diff
@@ -21,17 +21,20 @@ sensor:
   id: 1
 ```
 
+As of Feb 2020 including the option `use_emoncms_unit` will discover all sensors from Emoncms and use the unit of measurement specified in the Feed in Emoncms.  Tested with [Emoncms](https://github.com/emoncms/emoncms) V10.1.13.
+
 ## Configuration variables
```

Comment:
I see that the way in which the config variables are described here is totally inconsistent with how we handle it today. I assign myself to this PR to fix this.